setup.py

```diff
@@ -1,31 +1,124 @@
-import setuptools
+"""
+jupyter-server-proxy setup
+"""
+import json
 from glob import glob
+from pathlib import Path
+
+import setuptools
+from jupyter_packaging import (
+    combine_commands,
+    create_cmdclass,
+    ensure_targets,
+    install_npm,
+    skip_if_exists,
+)
+
+HERE = Path(__file__).parent.resolve()
+
+# The name of the project
+name = "jupyter_server_proxy"
+
+lab_path = HERE / name / "labextension"
+
+# Representative files that should exist after a successful build
+jstargets = [
+    str(lab_path / "package.json"),
+]
+
+package_data_spec = {
+    name: ["*"],
+}
+
+labext_name = "jupyter_server_proxy"
```

Comment:
```suggestion
labext_name = "@jupyterlab/server-proxy"
```
This fixes the labextension for me

Reply:
Thanks a lot for your help!

Reply:
Can confirm that it's also working for me.